host-configs/ORNL/frontier-base.cmake

```diff
@@ -23,7 +23,7 @@ set( GEOSX_BUILD_OBJ_LIBS OFF CACHE BOOL "" FORCE )
 set( ENABLE_GTEST_DEATH_TESTS OFF CACHE BOOL "" )
 set( gtest_disable_pthreads ON CACHE BOOL "" )
 
-set( ENABLE_TESTS OFF CACHE BOOL "" FORCE )
+set( GEOS_ENABLE_TESTS OFF CACHE BOOL "" FORCE )
```

Comment:
I am guessing they don't work on Frontier?

Reply:
They didn't reliably config / build for a long time, but neither do a ton of our kernels still. At some point I'll do a PR devoted to ensuring they all work now that Frontier is (relatively) stable